PostHog/Autocapture/SwiftUI/View+PostHogLabel.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  View+PostHogLabel.swift
+//  PostHog
+//
+//  Created by Yiannis Josephides on 04/12/2024.
+//
+
+#if os(iOS) || targetEnvironment(macCatalyst)
+    import SwiftUI
+
+    public extension View {
```

Comment:
can we combine modifiers? eg I want to add a label to a button but I don't want it to be captured in session recording